File: app/src/main/java/net/rmoreno/weatherapp/MyLocationListener.java
package net.rmoreno.weatherapp;

import android.location.Location;
import android.location.LocationListener;
import android.os.Bundle;
import android.util.Log;


public class MyLocationListener implements LocationListener{


    @Override
    public void onLocationChanged(Location location) {
        double latitude = location.getLatitude();
        double longitude = location.getLongitude();

        System.out.println("LATITUDE AND LONGITUDE" + latitude + "   " + longitude);
    }

    @Override
    public void onProviderEnabled(String provider) {
        Log.d("Latitude", "enable");
    }

    @Override
    public void onProviderDisabled(String provider) {
        Log.d("Latitude","disable");
    }

    @Override
    public void onStatusChanged(String provider, int status, Bundle extras) {
        Log.d("Latitude","status");
    }
}

File: app/src/main/java/net/rmoreno/weatherapp/MyLocationListener.kt
package net.rmoreno.weatherapp

import android.location.Location
import android.location.LocationListener
import android.os.Bundle
import android.util.Log


class MyLocationListener : LocationListener {
    override fun onLocationChanged(location: Location) {
        val latitude = location.latitude
        val longitude = location.longitude

        println("LATITUDE AND LONGITUDE$latitude   $longitude")
    }

    override fun onProviderEnabled(provider: String) {
        Log.d("Latitude", "enable")
    }

    override fun onProviderDisabled(provider: String) {
        Log.d("Latitude", "disable")
    }

    override fun onStatusChanged(provider: String, status: Int, extras: Bundle) {
        Log.d("Latitude", "status")
    }
}
